README.md

```diff
@@ -112,7 +112,7 @@ Please see [Using Loaders](https://webpack.github.io/docs/using-loaders.html) fo
 **styling**
 * [`style`](https://github.com/webpack/style-loader): Add exports of a module as style to DOM
 * [`css`](https://github.com/webpack/css-loader): Loads css file with resolved imports and returns css code
-* [`cssnext`](https://github.com/MoOx/cssnext-loader): Loads and compiles a css file using [cssnext](http://cssnext.io/)
+* [`cssnext`](https://github.com/MoOx/postcss-cssnext): Loads and compiles a css file using [cssnext](http://cssnext.io/)
```

Comment:
Actually this might be a better idea, possibly with a link in the text to the available plugins? - https://github.com/postcss/postcss#plugins